docs/instructor/autograde.md

```diff
@@ -41,36 +41,38 @@ This checklist ensures that you have implemented key technical and business logi
 
 ### [Container Output](#container-output-details)
 
-- [ ] Your container logic places any output data in the `/output` path following this convention:
+- [ ] Your container logic places any output data, if it needs to persist, in the `/output` path following this convention:
   `/output/admin/`
   `/output/staff/`
   `/output/student/`
 - [ ] `/output/staff` sub-directory contains:
-  - `report.json` grading file at the end of a grading run
-  - additional files that TAs and instructors need access to after the grading run
+  - `report.json` grading file at the end of a successful grading run that exits with a success `0` exit code.
+  - additional files that TAs and instructors need access to after the grading run.
 - [ ] The `report.json` file is valid JSON that follows the [Report Schema](https://github.com/ubccpsc/classy/blob/master/packages/common/types/ContainerTypes.ts#L72-L115).
-- [ ] Your container logic assumes that if code stalls, encounters an infinite loop, or the container times out, Classy will provide this [default report.json file](https://github.com/ubccpsc/classy/blob/master/packages/autotest/src/autotest/GradingJob.ts#L28-L40).
+- [ ] References to attachments are recorded in the `report.json` file under the `attachments` attribute.
```

Comment:
Still not sure how to use attachments.

Reply:
Just added a bit more info for you @wilbowma .